src/data/battle_moves.h

```diff
@@ -3491,39 +3245,27 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_DYNAMAX] =
 
     [MOVE_GIGA_DRAIN] =
     {
-        #if B_UPDATED_MOVE_DATA >= GEN_5
-            .power = 75,
-            .pp = 10,
-        #elif B_UPDATED_MOVE_DATA == GEN_4
-            .power = 60,
-            .pp = 10,
-        #else
-            .power = 60,
-            .pp = 5,
-        #endif
         .effect = EFFECT_ABSORB,
+        .power = B_UPDATED_MOVE_DATA >= GEN_5 ? 75 : 60,
         .type = TYPE_GRASS,
         .accuracy = 100,
+        .pp =  B_UPDATED_MOVE_DATA >= GEN_4 ? 10 : 5,
```

Comment:
Whitespace after the equation mark.

Reply:
There is?
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/4e36298f-9580-4470-8b03-f7df022061da)

Reply:
pp =(2SPACE)B_UPDATED... no other one has 2 spaces.

Reply:
Fixed